src/components/MovieList.js

```diff
@@ -43,66 +40,29 @@ const Container = styled.div`
 `;
 
 const MovieHeadline = styled.p`
-	padding: 8px 8px 16px 12px;
+	min-height: 56px;
+	padding: 12px 8px 16px 12px;
 	color: black;
 	overflow-wrap: break-word;
 `;
 
-export default function MovieList() {
-	const arrData = useStore(state => state.arrData);
-	const setData = useStore(state => state.setData);
-	const arrConfiguration = useStore(state => state.arrConfiguration);
-	const hasLoadingErrorOccured = useStore(state => state.hasLoadingErrorOccured);
-	const setLoadingErrorOccured = useStore(state => state.setLoadingErrorOccured);
-
-	const imagesBaseUrl = arrConfiguration.base_url + arrConfiguration.poster_sizes[1];
-
-	const API_KEY = process.env.API_KEY;
-
-	function fetchUrl(url) {
-		setLoadingErrorOccured(false);
-		fetch(url)
-			.then(res => res.json())
-			.then(data => {
-				setData(data.results);
-			})
-			.catch(function (error) {
-				console.log(error);
-				setLoadingErrorOccured(true);
-				setData([]);
-			});
-	}
-
-	function loadMovies() {
-		fetchUrl(
-			`https://api.themoviedb.org/3/discover/movie?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`
-		);
-	}
-	function loadTVShows() {
-		fetchUrl(
-			`https://api.themoviedb.org/3/discover/tv?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`
-		);
-	}
+export default function MovieList(data) {
+	//
+	const arrData = data.data;
```

Comment:
This is a bad name for the `props` array that we pass to React components. Either name it `props` or use destructuring assignment here to extract data.
```suggestion
export default function MovieList({ data }) {
```
That way you also don't need the following line anymore:
```js
	const arrData = data.data;
```

Reply:
renamed.